packages/serverless-offline-sqs/src/index.js

```diff
@@ -32,15 +30,17 @@ const defaultOptions = {
 const omitUndefined = omitBy(isUndefined);
 
 class ServerlessOfflineSQS {
-  constructor(serverless, cliOptions) {
+  constructor(serverless, cliOptions, {log}) {
     this.cliOptions = null;
     this.options = null;
+    this.log = null;
     this.sqs = null;
     this.lambda = null;
     this.serverless = null;
 
     this.cliOptions = cliOptions;
     this.serverless = serverless;
+    this.log = log;
```

Comment:
log could be undefined here, don't want it to break 


```suggestion
this.log = log || {
  debug: console.debug.bind(console),
  notice: console.log.bind(console),
  warning: console.warn.bind(console)
};```